README.md

```diff
@@ -229,9 +229,11 @@ Auth0
     }
 ```
 
-#### Disable Single Sign On (iOS 13+ / macOS)
+#### Disable Single Sign On Consent Alert (iOS 13+ / macOS)
 
-Add the `useEphemeralSession()` method to the chain to disable SSO on iOS 13+ and macOS. This way the system will not display the consent popup that otherwise shows up when SSO is enabled. It has no effect on older versions of iOS.
+Add the `useEphemeralSession()` method to the chain to disable SSO on iOS 13+ and macOS. This way the system will not display the consent alert that otherwise shows up when SSO is enabled. It has no effect on older versions of iOS.
```

Comment:
I know this hasn't changed, but after having another read of this, I wonder if we need to do a better job of highlighting the trade-off. Yes, we can get rid of the alert but the downside is you won't be able to use SSO.

I would reword this a little and link off to something that helps explain the impact:

```suggestion
To suppress the alert box, add the `useEphemeralSession()` method to the chain. This has the impact of disabling [Single Sign On (SSO)](https://auth0.com/docs/sso) on iOS 13+ and macOS, but will also not display the consent alert that otherwise shows up when SSO is enabled. It has no effect on older versions of iOS.
```

Just a suggestion, and there might be a better link, but you get the idea.